test/unit/FundMe.test.ts

```diff
@@ -1,153 +1,222 @@
-        deployer.address
-      )
-
-      // Assert
-      assert.equal(endingFundMeBalance.toString(), "0")
-      assert.equal(
-        startingFundMeBalance.add(startingDeployerBalance).toString(),
-        endingDeployerBalance.add(gasCost).toString()
-      )
-    })
-    // this test is overloaded. Ideally we'd split it into multiple tests
-    // but for simplicity we left it as one
-    it("is allows us to withdraw with multiple funders", async () => {
-      // Arrange
-      const accounts = await ethers.getSigners()
-      await fundMe
-        .connect(accounts[1])
-        .fund({ value: ethers.utils.parseEther("1") })
-      await fundMe
-        .connect(accounts[2])
-        .fund({ value: ethers.utils.parseEther("1") })
-      await fundMe
-        .connect(accounts[3])
-        .fund({ value: ethers.utils.parseEther("1") })
-      await fundMe
-        .connect(accounts[4])
-        .fund({ value: ethers.utils.parseEther("1") })
-      await fundMe
-        .connect(accounts[5])
-        .fund({ value: ethers.utils.parseEther("1") })
-      // Act
-      const startingFundMeBalance = await fundMe.provider.getBalance(
-        fundMe.address
-      )
-      const startingDeployerBalance = await fundMe.provider.getBalance(
-        deployer.address
-      )
-      const transactionResponse = await fundMe.cheaperWithdraw()
-      // Let's comapre gas costs :)
-      // const transactionResponse = await fundMe.withdraw()
-      const transactionReceipt = await transactionResponse.wait()
-      const { gasUsed, effectiveGasPrice } = transactionReceipt
-      const withdrawGasCost = gasUsed.mul(effectiveGasPrice)
-      console.log(`GasCost: ${withdrawGasCost}`)
-      console.log(`GasUsed: ${gasUsed}`)
-      console.log(`GasPrice: ${effectiveGasPrice}`)
-      const endingFundMeBalance = await fundMe.provider.getBalance(
-        fundMe.address
-      )
-      const endingDeployerBalance = await fundMe.provider.getBalance(
-        deployer.address
-      )
-      // Assert
-      assert.equal(
-        startingFundMeBalance.add(startingDeployerBalance).toString(),
-        endingDeployerBalance.add(withdrawGasCost).toString()
-      )
-      await expect(fundMe.s_funders(0)).to.be.reverted
-      assert.equal(
-        (await fundMe.s_addressToAmountFunded(accounts[1].address)).toString(),
-        "0"
-      )
-      assert.equal(
-        (await fundMe.s_addressToAmountFunded(accounts[2].address)).toString(),
-        "0"
-      )
-      assert.equal(
-        (await fundMe.s_addressToAmountFunded(accounts[3].address)).toString(),
-        "0"
-      )
-      assert.equal(
-        (await fundMe.s_addressToAmountFunded(accounts[4].address)).toString(),
-        "0"
-      )
-      assert.equal(
-        (await fundMe.s_addressToAmountFunded(accounts[5].address)).toString(),
-        "0"
-      )
-    })
-  })
-})
+import { ethers, deployments, network } from 'hardhat'
```

Comment:
This change is too much for me to review. It looks like you've changed the whole file.